main.go

```
@@ -60,48 +58,13 @@ func main() {
-	return &cobra.Command{
-		Use:   "scrapper",
-		Short: "Scrapper Supreme Court of Singapore",
-		Long:  "Scrapper Supreme Court of Singapore",
-		Run: func(cmd *cobra.Command, args []string) {
-
-			scrapper.StartScraper()
-		},
-	}
+	scrapper := scrapper.ScrapperImpl{}
+	scrapper.Setup()
+	scrapper.ScrapeAll(ctx)
```

golangci-lint (1.62.2)

Error return value of `scrapper.ScrapeAll` is not checked

(errcheck)